pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/OpphoererBarnetilleggAuto.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.pensjon.brev.api.model.maler
+
+import no.nav.pensjon.brev.api.model.Kroner
+import no.nav.pensjon.brev.api.model.Sivilstand
+import no.nav.pensjon.brev.api.model.vedlegg.MaanedligUfoeretrygdFoerSkattDto
+import no.nav.pensjon.brev.api.model.vedlegg.OpplysningerBruktIBeregningUTDto
+import no.nav.pensjon.brev.api.model.vedlegg.OrienteringOmRettigheterUfoereDto
+import java.time.LocalDate
+
+@Suppress("unused")
+data class OpphoererBarnetilleggAutoDto(
+    val foedselsdatoPaaBarnetilleggOpphoert: LocalDate,
+    val oensketVirkningsDato: LocalDate,
+    val barnetilleggFellesbarn: BarnetilleggFellesbarn?,
+    val barnetilleggSaerkullsbarn: BarnetilleggSaerkullsbarn?,
+    val brukerBorInorge: Boolean,
+    val grunnbeloep: Kroner,
+    val sivilstand: Sivilstand,
+    val ufoeretrygd: Ufoeretrygd,
+    val maanedligUfoeretrygdFoerSkatt: MaanedligUfoeretrygdFoerSkattDto,
+    val opplysningerBruktIBeregningUT: OpplysningerBruktIBeregningUTDto,
+    val orienteringOmRettigheterUfoere: OrienteringOmRettigheterUfoereDto,
+)
+
+data class Ufoeretrygd(
+    val ufoertrygdUtbetalt: Int,
```

Comment:
Skal vi få laget en standard for om vi skriver før/etter avkorning, utbetalt/før skatt eller netto/brutto? Jeg har brukt netto/brutto så langt.